src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,126 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private StorageDao storageDao;
+    private RegistrationServiceImpl registrationService;
+
+    @BeforeEach
+    void setUp() {
+        storageDao = mock(StorageDao.class);
+        registrationService = new RegistrationServiceImpl(storageDao);
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        assertThrows(RegistrationException.class, () -> registrationService.register(null),
+                "User cannot be null");
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        User user = new User();
+        user.setLogin(null);
+        user.setPassword("password123");
+        user.setAge(20);
+
+        assertThrows(RegistrationException.class, () -> registrationService.register(user),
+                "Login must be at least 6 characters long");
```

Comment:
The exception message 'Login must be at least 6 characters long' does not match the test case. The test case is for a null login, not a short login. The message should be informative about the actual reason for the failure, such as 'Login cannot be null'.